packages/discord.js/src/structures/ThreadChannel.js

```diff
@@ -456,6 +456,27 @@ class ThreadChannel extends BaseChannel {
   setAppliedTags(appliedTags, reason) {
     return this.edit({ appliedTags, reason });
   }
+  
+  /**
+   * Pins this thread channel from the forum channel.
```

Comment:
Consistent suggestion as "thread channel" is written twice in the documentation:
```suggestion
   * Pins this thread from the forum channel.
```